nni/retiarii/execution/base.py

```diff
@@ -1,12 +1,15 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
+from __future__ import annotations
```

Comment:
Add an empty line before this.
https://peps.python.org/pep-0008/#module-level-dunder-names